module_4/gift_certificates_system/model/src/main/java/com/epam/esm/model/util/DtoRegexValidator.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.model.util;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class DtoRegexValidator {
+    // User DTO
+    public static final String USER_LOGIN = "^[\\p{Ll}\\p{Lu}]{4,50}$";
+    public static final String USER_NAME = "^\\p{Lu}\\p{Ll}{1,50}$";
+
+    // Tag DTO
+    public static final String TAG_NAME = "^\\p{Lu}\\p{Ll}{1,50}$";
+
+    // Gift certificate DTO
```

Comment:
Please remove comments, everything is clear from the naming

Reply:
Thanks, was [removed](https://github.com/n199a/epam_training_center/pull/4/commits/8d4ce0b980d1aea206cf45781d79f362b3581b15).